code/game/machinery/recycler.dm

```diff
@@ -160,10 +164,6 @@
 		playsound(src, 'sound/machines/buzz-sigh.ogg', (50 + not_eaten*5), FALSE, not_eaten, ignore_walls = (not_eaten - 10)) // Ditto.
 	if(!ismob(morsel))
 		qdel(morsel)
-	else // Lets not qdel a mob, yes?
-		for(var/iterable in morsel.contents)
-			var/atom/movable/content = iterable
-			qdel(content)
```

Comment:
I'm spooked about removing this code, but `/datum/unit_test/human_through_recycler` works fine (presumably because we already got all of the "morsel's" contents above and didn't have to get it again) so w/e